.github/workflows/deploy.yml

```
@@ -0,0 +1,75 @@
+  pull_request:
+    branches: [ "develop" ] # PR 생성 시 빌드/테스트 진행
+  workflow_dispatch:    # 수동 가능하게
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      # 코드 가져오기
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)